src/main/java/seedu/exchangecoursemapper/command/PersonalTrackerCommand.java

```diff
@@ -0,0 +1,24 @@
+package seedu.exchangecoursemapper.command;
+
+import seedu.exchangecoursemapper.storage.Storage;
+
+import javax.json.Json;
+import javax.json.JsonObject;
+import javax.json.JsonReader;
+import java.io.IOException;
+import java.io.InputStream;
+
+public abstract class PersonalTrackerCommand {
+    public static final String FILE_PATH = "/database.json";
+
+    public JsonObject createJsonObject() throws IOException {
+        InputStream inputStream = getClass().getResourceAsStream(FILE_PATH);
+        if (inputStream == null) {
+            throw new IOException();
+        }
+        JsonReader jsonReader = Json.createReader(inputStream);
+        return jsonReader.readObject();
+    }
```

Comment:
Might have violated DRY as this method can be found in `Command.java` in `command` package